build.gradle

```diff
@@ -384,7 +384,11 @@ task integTestRemote(type: RestIntegTestTask) {
     }
 }
 
-String bwcVersion = "1.1.0.0"
+String bwcMinVersion = "1.1.0.0"
+String bwcBundleVersion = "1.3.2.0"
```

Comment:
Is the standard supposed to be the oldest version from the previous major version or the latest? IMO we should only need to persist one or the other rather than both here.

Reply:
Hi @ohltyler The reason I use 1.3.2 here is this is the earliest version that we start to support with our current URL pattern. For example, `https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/1.3.2/latest/linux/x64/tar/dist/opensearch/opensearch-1.3.2-linux-x64.tar.gz`. We introduced an additional path for different distribution types since then. 
I kept the old `1.1` for plugin level testing to minimize the effects of this PR on your original tests but I'm ok to change that to 1.3.2.0 as well.

Reply:
Got it - I think it's fine to leave as-is. Just to confirm, in general, we want to use the earliest version, right? Like for example, in `main`, we should point to `2.0.0.0` as the BWC version used for upgrading to `3.x`?

Reply:
Ideally yes I believe. In future we should have a matrix of versions and test each of the versions on upgrading and etc.. For now, i don't see any enforcement on the BWC version for `main`, it also depends on the core repo.